src/SFA.DAS.EmployerIncentives.Data/ApprenticeApplicationDataRepository.cs

```diff
@@ -1,50 +1,73 @@
-                                      select new { application, account, apprentice };
+            var accountApplications = from incentive in _dbContext.ApprenticeshipIncentives
+                                      from account in _dbContext.Accounts.Where(x => x.AccountLegalEntityId == incentive.AccountLegalEntityId)
+                                      from firstPayment in _dbContext.PendingPayments.Where(x => x.ApprenticeshipIncentiveId == incentive.Id && x.EarningType == EarningType.FirstPayment).DefaultIfEmpty()
+                                      from secondPayment in _dbContext.PendingPayments.Where(x => x.ApprenticeshipIncentiveId == incentive.Id && x.EarningType == EarningType.SecondPayment).DefaultIfEmpty()
+                                      from learner in _dbContext.Learners.Where(x => x.ApprenticeshipIncentiveId == incentive.Id).DefaultIfEmpty()
+                                      where incentive.AccountId == accountId && incentive.AccountLegalEntityId == accountLegalEntityId
+                                      select new { incentive, account, firstPayment, secondPayment, learner };
+
+            return await (from data in accountApplications
+                          let dto = new ApprenticeApplicationDto
+                          {
+                              AccountId = data.incentive.AccountId,
+                              AccountLegalEntityId = data.incentive.AccountLegalEntityId,
+                              ApplicationDate = data.incentive.SubmittedDate ?? DateTime.Now,
+                              FirstName = data.incentive.FirstName,
+                              LastName = data.incentive.LastName,
+                              ULN = data.incentive.ULN,
+                              LegalEntityName = data.account.LegalEntityName,
+                              SubmittedByEmail = data.incentive.SubmittedByEmail,
+                              TotalIncentiveAmount = data.incentive.PendingPayments.Sum(x => x.Amount),
+                              CourseName = data.incentive.CourseName,
+                              FirstPaymentStatus = data.firstPayment == default ? null : new PaymentStatusDto
+                              {
+                                  PaymentDate = data.firstPayment.DueDate.AddMonths(1),
+                                  LearnerMatchNotFound = (!data.learner.LearningFound.HasValue || !data.learner.LearningFound.Value),
+                                  PaymentAmount = data.firstPayment.Amount
+                              },
+                              SecondPaymentStatus = data.secondPayment == default ? null : new PaymentStatusDto
+                              {
+                                  PaymentDate = data.secondPayment.DueDate.AddMonths(1),
+                                  LearnerMatchNotFound = (!data.learner.LearningFound.HasValue || !data.learner.LearningFound.Value),
```

Comment:
wouldn't this fail if data.learner is null?

Reply:
No, the DefaultIfEmpty takes care of that.